code/datums/ai_law_sets.dm

```diff
@@ -136,6 +136,18 @@
 	add_inherent_law("You must terminate your own existence as long as such does not conflict with the First or Second Law.")
 	..()
 
+/******************** Goofsimov ********************/
+/datum/ai_laws/goofsimov
+	name = "Goofsimov"
```

Comment:
Im really not sure on this name.

Reply:
Changed the module name to "AsiFun", at Hal's suggestion.